### PR TITLE
Increased ts eslint strictness

### PR DESCRIPTION
**Pull request by Vithop**

### CODE REVIEW

File: svelte/src/lib/Game.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Game {
   #boardState: string[];
   constructor(
     whosTurn: GameStatus = Status.XTurn,
-    boardState = Array(9).fill("")
+    boardState: string[] = new Array<string>(9).fill("")
```
Comment (Mathuran) [inline svelte/src/lib/Game.ts:48]: So verbose now. Kotlin is nicer to read.

**Review by Mathuran — APPROVED** [review]
Is this change thread safe 🤔

File: svelte/src/lib/Game.ts (hunk 1 above)
Comment (Khufu-I) [reply svelte/src/lib/Game.ts:48]: Very nice, Typescript is finally working. On the Kotlin side there is a `Board` type which is underpinned by `Bitboard`. Probably would be good to have a similar implementation on the Javascript side.

**Review by Khufu-I — APPROVED** [review]

File: svelte/src/lib/Game.ts (hunk 1 above)
Comment (Vithop) [reply svelte/src/lib/Game.ts:48]: It's thread safe tho

**Pull request merged**